apps/mark-scan/frontend/src/pages/admin_screen.tsx

```diff
@@ -62,6 +58,18 @@ export function AdminScreen({
 }: AdminScreenProps): JSX.Element {
   const { election } = electionDefinition;
   const logOutMutation = logOut.useMutation();
+  const getPrecinctSelectionQuery = getPrecinctSelection.useQuery();
+  const precinctSelection = getPrecinctSelectionQuery.data?.ok();
+
+  const setPrecinctSelectionMutation = setPrecinctSelection.useMutation();
+  const updatePrecinctSelection = useCallback(
+    (newPrecinctSelection: PrecinctSelection) => {
+      setPrecinctSelectionMutation.mutate({
+        precinctSelection: newPrecinctSelection,
+      });
+    },
+    [setPrecinctSelectionMutation]
```

Comment:
Mutation objects aren't memoized, so I'm pretty sure this `useCallback` is unnecessary because it will recreate the function on every render anyway.

Reply:
Updated in https://github.com/votingworks/vxsuite/pull/3786/files#diff-eba06bf39f2aae2ca6cf5b42615541312f20619de8c93b109f2eaa45a134fff5L63